src/components/safe-apps/SafeAppsErrorBoundary/SafeAppsLoadError.tsx

```diff
@@ -0,0 +1,35 @@
+import { useRouter } from 'next/router'
+import Typography from '@mui/material/Typography'
+import Link from '@mui/material/Link'
+import Button from '@mui/material/Button'
+import OpenInNew from '@mui/icons-material/OpenInNew'
+
+import css from './styles.module.css'
+
+const SafeAppsLoadError = (): React.ReactElement => {
+  const router = useRouter()
+
+  return (
+    <div className={css.wrapper}>
+      <div className={css.content}>
+        <Typography variant="h1">Safe App could not be loaded</Typography>
+
+        <img src="/images/network-error.svg" alt="Safe Apps load error" className={css.image} />
+
+        <div>
+          <Typography component="span">In case the problem persists, please reach out to us via </Typography>
+          <Link target="_blank" href="https://chat.gnosis-safe.io" fontSize="medium">
```

Comment:
Please move `https://chat.gnosis-safe.io` to constants.

Reply:
done